assets/javascripts/src/modules/abTests.es6

```diff
@@ -8,11 +8,9 @@ import { SET_AMOUNT } from 'src/actions';
 export function init() {
     const state = store.getState();
 
-    ophan.loaded.then(function (ophan) {
-        ophan.record({
-            abTestRegister: abTestData(state.data.abTests, false)
-        })
-    });
+    registerTestsWithOphan(state.data.abTests, initTestFlags);
```

Comment:
I'm not sure this really needs the second parameter. At this point it's fine to just send `complete: false` for all of the active tests. Sending `complete: true` is down to each individual test, and might happen at any point in the app's lifecycle so it should be enough to just provide a helper function that fires the `true` event for a test.